de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java

```diff
@@ -127,56 +120,31 @@ private String writeToAndAssert() throws WebApplicationException, IOException {
 
 	// Test the marshalling
 	@Test
-	public void testWriteTo() throws WebApplicationException, IOException {
-		writeToAndAssert();
-	}
-	
-	/**
-	 * Open the remote repository and count the commits
-	 * @return number of commits in remote
-	 * @throws IOException
-	 * @throws NoHeadException
-	 * @throws GitAPIException
-	 * @throws InterruptedException
-	 */
-	private int countCommits() throws IOException, NoHeadException, GitAPIException, InterruptedException {
-		Git git = Git.open(pathRepoRemote.toFile());
-		Iterator<RevCommit> iterator = git.log().call().iterator();
-		int commits = 0;
-		while (iterator.hasNext()) {
-			iterator.next();
-			commits++;
-		}
-		git.close();
-		return commits;
+	public void testWriteTo() throws WebApplicationException, IOException, NoHeadException, GitAPIException, InterruptedException {
+		writeToAndAssert(testString);
 	}
 	
 	// Test the unmarshalling
 	@SuppressWarnings("unchecked")
 	@Test
 	public void testReadFrom() throws WebApplicationException, IOException, NoHeadException, GitAPIException, InterruptedException {
-
-		int initialCommits = countCommits();
 		
-		String output = writeToAndAssert();
+		String output = writeToAndAssert(testString);
 		
-		// No changes don't create a commit
+		// No changes
```

Comment:
Comment here could use some expansion i.e. "Write did not cause any changes" otherwise it's not clear what the no changes are referring to (data or the code istelf).